packages/compartment-mapper/src/parse-cjs-shared-export-wrapper.js

```diff
@@ -8,6 +8,47 @@ const { freeze, keys, create, hasOwnProperty } = Object;
  * @returns {boolean}
  */
 const has = (object, key) => apply(hasOwnProperty, object, [key]);
+const removeProtocol = url => {
+  if (url.substring(0, 7) === 'file://') {
+    url = url.substring(7);
+  }
+  return url;
+};
+const noTrailingSlash = path => path.replace(/[\\/]$/, '');
```

Comment:
I don’t feel strongly about avoiding a regex here, but I am not strong enough with regexen to tell you whether they’re ReDoS risks. I assume this compiles to a regex that doesn’t scan the whole length of the string.